app/views/documents/show/_topics.html.erb

```diff
@@ -1,13 +1,13 @@
 <% begin %>
   <% service = TopicsService.new %>
-  <% topics = service.topics_for_document(@document.content_id) %>
+  <% topic_content_ids, _version = service.topics_for_document(@document) %>
 
   <% breadcrumbs = capture do %>
-    <% if topics.any? %>
+    <% if topic_content_ids.any? %>
       <div class="topic-breadcrumb">
-        <% topics.each do |topic| %>
+        <% topic_content_ids.each do |topic_content_id| %>
           <ol>
-            <% service.topic_breadcrumb(topic).reverse.each do |crumb| %>
+            <% service.topic_breadcrumb(topic_content_id).reverse.each do |crumb| %>
```

Comment:
Do we reverse this because this method returns results in a [child, parent, grandparent, great-grandparent] fashion? If so I wonder if we'd want to reverse the method as I'd suggest breadcrumbs suggests the inverse.

Reply:
Good spot - I think we did it like this originally because the method was called something else. Fixed :-).